File: AMS_电子对账/code/client/source/YssElecReco/Service/Er/IErDataService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using FAST.Common.Service.Services.Base;
using YssInformation.Support.Sys.Dictionary.Pojo;
using FAST.Common.Service.Attributes;
using YssElecReco.Context;

namespace YssElecReco.Service.Er
{
    /// <summary>
    /// 产生电子对账核对数据service
    /// chenyoulong 20121213
    /// v1.0.0.6
    /// </summary>
    [ServiceAttribute(ServiceId = ElecrecoServiceIdConstant.OSGI_ELECRECO, ServiceController = "com.yss.uco.elecreco.er.erdata.controller.IErDataServiceController")]
    public interface IErDataService : IServiceBus
    {
        /// <summary>
        /// 获取对账类型数据
        /// </summary>
        /// <returns>对账类型字典list</returns>
        [MethodAttribute]
        List<DzType> getDzType();
    }
}

File: AMS_电子对账/code/server/source/com.yss.uco.elecreco/src/main/java/com/yss/uco/elecreco/er/erdata/service/IErDataService.java
package com.yss.uco.elecreco.er.erdata.service;
import com.yss.framework.api.mvc.biz.IServiceBus;
import com.yss.framework.api.restful.annotations.RestfulSupported;
import com.yss.uco.elecreco.support.dzdz.common.pojo.XmlFile;

@RestfulSupported
public interface IErDataService extends IServiceBus{

	/**
	 * 保存xml文件
	 * @param fsn
	 * 返回文件内容
	 */
	public XmlFile getXmlFileRoot(String fsn,String fileType,String cAssCode);
}
